### ISSUE TITLE
Save inbound email attachments to S3

### ISSUE THREAD

**Issue by thinkingserious** [sendgrid/sendgrid-csharp#558]
When emails come to inbound parse, sometimes they have attachments
It would be nice to be able to send the files to an S3 bucket when they are received

**Comment by kylegalbraith** [comment]
@thinkingserious Where would this need to happen? Where are inbound email attachments currently saved?

**Comment by thinkingserious** [comment]
Hello @kylegalbraith,

This issue is dependent on #328

**Comment by thinkingserious** [comment]
Since there has been no activity on this issue since March 1, 2020, we are closing this issue. Please feel free to reopen or create a new issue if you still require assistance. Thank you!
